test/dotnet-run.Tests/GivenDotnetRunRunsCsProj.cs

```diff
@@ -75,6 +75,25 @@ public void ItCanRunAMultiTFMProjectWithImplicitRestore()
                          .And.HaveStdOutContaining("This string came from the test library!");
         }
 
+        [Fact]
```

Comment:
Should we add a test for the verbosity option?

Reply:
I'll amend the commit with one.

Reply:
Added a test to that commit to check that the verbosity option does something.  Waiting on an issue to reference, if there is an existing one, before pushing it up.

Reply:
Pushed in ea85555.